lib/linters/cfn-lint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Can this script look for .cfnlintrc as per https://github.com/awslabs/cfn-python-lint?